### DISCUSSION TITLE
audio quality

### DISCUSSION THREAD

**Discussion by shivaraj65** [pear-devs/pear-desktop#860] [Ideas]
Its a really good project, I have to congratulate you guys for that. Moving on to the topic, It would be great to have high res audio (high audio quality) like the vanced music app has. I aint comparing your app with others. I am just saying high res audio would be a nice touch and elevate the music listening experience further.
<img width="960" alt="12213" src="https://user-images.githubusercontent.com/51598598/196029361-df773fdf-27f2-4afa-9cd2-152c0eb19cd1.png">

**Comment by Bellisario** [comment]
I don’t know if that’s possible, because audio is directly managed by YouTube Music and not by the app…

**Comment by Liknox** [comment]
Youtube Music as well allow you only 2 choices ( between low and normal ), so it's impossible to add high audio quality.
